include/Pickup.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "Definitions.h"
+#include "Access.h"
+#include "GameSpecificComponents.h"
+
+struct PickupSystem
+	: public ecs::System
+	, public MutAccessGroupStorage<Player, Position>
+	, public MutAccessGroupStorage<Pickup, Position>
+{
+	using QueryPlayers 	= MutAccessGroupStorage<Player, Position>;
```

Comment:
weird spacing, no need to align or anything like that, let your IDE do the hard work :)